docs/concepts/scheduler.md

```diff
@@ -86,6 +87,16 @@ If provided, the Gardener Scheduler will only consider those seeds as "suitable"
 By default only seeds with the same provider than the shoot are selected. By adding a `providerTypes` field to the `seedSelector`
 a dedicated set of possible providers (`*` means all provider types) can be selected.
 
+## Ensuring seeds capacity for shoots is not exceeded
+
+Seeds have a practical limit of how many shoots they can accommodate. Exceeding this limit is undesirable as the system performance will be noticeably impacted. Therefore, the scheduler ensures that a seed's capacity for shoots is not exceeded by taking into account a maximum number of shoots that can be scheduled onto a seed.
+
+This mechanism works as follows:
+
+* The `gardenlet` is configured with certain *resources* and their total *capacity* (and, for certain resources, the amount *reserved* for Gardener), see [/example/20-componentconfig-gardenlet.yaml](../../example/20-componentconfig-gardenlet.yaml). Currently, the only such resource is the maximum number of shoots that can be scheduled onto a seed. 
+* The `gardenlet` seed controller updates the `capacity` and `allocatable` fields in Seed status with the capacity of each resource and how much of it is actually available to be consumed by shoots. 
```

Comment:
Most people with k8s background will probably understand this right away, but can you still add an explanation, that `allocatable` is basically just `capacity - reserved` (so a static value depending on the configuration)?

Reply:
Done